packages/muelu/adapters/amgx/MueLu_AMGXOperator_decl.hpp

```diff
@@ -241,7 +242,7 @@ namespace MueLu {
 #ifdef NEW_COMM
         AMGX_resources_create(&Resources_, Config_, &mpiComm, 1/* number of GPU devices utilized by this rank */, device);
 #else
-        AMGX_resources_create(&Resources_, Config_, MPI_COMM_WORLD, 1/* number of GPU devices utilized by this rank */, device);
+        AMGX_resources_create(&Resources_, Config_, MueLu::GetGlobalComm(), 1/* number of GPU devices utilized by this rank */, device);
 #endif
```

Comment:
@csiefer2 Would it make more sense to remove the second code path since it's not compiled anyway?

Reply:
If we were to remove it or use `Teuchos::DefaultComm` here, then I'd just remove the newly added `MueLu::GlobalComm` stuff. 
What do you think?